server/src/test/java/org/elasticsearch/index/mapper/DateFieldTypeTests.java

```diff
@@ -53,67 +51,46 @@
 import org.elasticsearch.index.query.QueryRewriteContext;
 import org.elasticsearch.index.query.QueryShardContext;
 import org.joda.time.DateTimeZone;
-import org.junit.Before;
 
 import java.io.IOException;
 import java.time.Instant;
 import java.time.ZoneOffset;
+import java.util.Collections;
+import java.util.Map;
 
 public class DateFieldTypeTests extends FieldTypeTestCase<DateFieldType> {
-    @Override
-    protected DateFieldType createDefaultFieldType() {
-        return new DateFieldType();
-    }
 
-    @Before
```

Comment:
Formatter and Resolution are non-modifiable, so the tests on DateFieldMapperTests replace these bits